src/Controller/Crudit/TemplateController.php

```diff
@@ -40,9 +45,39 @@ public function duplicate(Template $template): Response
             $this->em->persist($copyTemplate);
             $this->em->flush();
         } else {
-            $this->addFlash('danger', $this->translator->trans('flash.copyAlreadyExist', [], 'LleHermesBundle'));
+            $this->addFlash(
+                FlashBrickResponse::ERROR,
+                $this->translator->trans('flash.copyAlreadyExist', [], 'LleHermesBundle')
+            );
         }
 
         return $this->redirectToRoute('lle_hermes_crudit_template_index');
     }
+
+
+    #[Route('/copy-for-tenant/{id}', name:'lle_hermes_crudit_template_copyfortenant', methods:['GET'])]
+    public function copyForTenant(Template $template, Request $request): Response
+    {
+        /** @var class-string $tenantClass */
+        $tenantClass = $this->parameterBag->get('lle_hermes.tenant_class');
```

Comment:
You have a service that manages this, you can add a `getTenantClass()` method into it